<launch>
	<rosparam command="load" file="$(find composit_planner)/launch/chem_params.yaml" />
	<rosparam command="load" file="$(find composit_planner)/launch/planner_params.yaml" />

	<node pkg="composit_planner" type="mcts_planner.py" name="mcts_planner" output="screen"/>
</launch>
